contracts/dezswap_pair/README.md

```diff
@@ -95,6 +93,36 @@ So, at a 1% tolerance level, if a user sends a transaction with 200 UST and 1 AS
   ```
 
 - Withdraw Liquidity (must be sent to liquidity token contract)
+
+  1. Without Min Assets
```

Comment:
The below JSON seems “with min assets” 🤔

Reply:
https://github.com/dezswap/dezswap-contracts/pull/6/commits/fddc2f8130a8b50b30284dddd3d42accf5bafe88